### PR TITLE
Admin merchant show #79

### LINKED ISSUE #79: User Story 38, Admin can see a merchant's dashboard
```
As an admin user
When I visit the merchant index page ("/merchants")
And I click on a merchant's name,
Then my URI route should be ("/admin/merchants/6")
Then I see everything that merchant would see
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #84

child of #125

### PR DESCRIPTION
**Pull request by Vjp888**
Implements access to a merchant's show page as an admin.

When they click the link on the merchants index it will redirect them to their showpage which shows all information about the merchant.

closes #79

### CODE REVIEW

File: spec/features/admin/merchant_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@admin)
+    end
+    it 'shows all the same information that a merchant would see' do
+      visit admin_merchants_path
+
+      within "#merchant-#{@merchant_1.id}" do
+        click_link "#{@merchant_1.name}"
+      end
+
+      expect(current_path).to eq("/admin/merchants/#{@merchant_1.id}")
```
Comment (wipegup) [inline spec/features/admin/merchant_show_spec.rb:18]: What about using a path helper here?

`admin_merchant_path(@merchant_1)`

**Review by wipegup — APPROVED** [review]

File: spec/features/admin/merchant_show_spec.rb (hunk 1 above)
Comment (whois-jon-peterson) [reply spec/features/admin/merchant_show_spec.rb:18]: I think this path would be `admin_merchants_path(@merchant_1)`

**Review by whois-jon-peterson — APPROVED** [review]
